baseband/dada/base.py

```diff
@@ -231,11 +231,11 @@ def __init__(self, fh_raw, thread_ids=None, squeeze=True):
         self._get_frame(0)
 
     @lazyproperty
-    def header1(self):
+    def _header_last(self):
```

Comment:
Now that we've gone for `stop_time`, can this become `_last_header` (and `last_frame` inside)?